packages/suite/src/views/dashboard/AssetsView/AssetTable/AssetTableHeader.tsx

```diff
@@ -1,43 +1,20 @@
-import styled from 'styled-components';
-
-import { spacingsPx, typography } from '@trezor/theme';
-
 import { Translation } from 'src/components/suite';
-import { AssetTableRowGrid } from './AssetTableRowGrid';
-
-const Header = styled.div`
-    display: flex;
-    ${typography.hint}
-    color: ${({ theme }) => theme.textSubdued};
-    align-items: center;
-    padding: ${spacingsPx.sm} 0;
-    border-bottom: 1px solid ${({ theme }) => theme.borderElevation2};
-
-    &:last-child {
-        padding-right: ${spacingsPx.md};
-    }
-`;
 
-const HeaderRight = styled(Header)`
-    justify-content: right;
-    padding-right: ${spacingsPx.xxxl};
-`;
+import { Table } from '@trezor/components';
 
 export const AssetTableHeader = () => (
-    <AssetTableRowGrid>
-        <Header /> {/* Logo */}
-        <Header>
+    <Table.Header>
+        <Table.Cell colSpan={3}>
```

Comment:
This is missing Table.Row. You probably deleted it because you don't want it to be highlighted? You can use the existing context hook from TableHeader to get info if the row is in header or not.

Reply:
Yes, I didn't want it to be affected by the styling, okay, thank you, I will use the hook